cloudify-web-monitoring-plugin/blueprints/AKS/README.md

```diff
@@ -0,0 +1,54 @@
+# Azure AKS Monitor Example
+
+## Prerequisites:
+
+  * Cloudify Manager with Latest Azure, Kubernetes, Utilities and Cloudify-web-monitoring Plugins.
+
+
+### Description:
+
+This example deploys AKS cluster and install a simple flask application that you can change the response time of URL,
+and create a monitor deployment that will measure HTTP server response time and will trigger a scale up workflow
+if that response time is above a threshold. The workflow will scale the cluster in AKS adding a worker to it.
+After the cooldown timer has passed and the low threshold is reached Cloudify will trigger a scale down workflow for AKS cluster.
+
+
+### Creating secrets
+
+Replace <value> with actual values, without the <>
+
+```shell
+cfy secrets create azure_subscription_id --secret-string <value>
+cfy secrets create azure_tenant_id --secret-string <value>
+cfy secrets create azure_client_id --secret-string <value>
+cfy secrets create azure_client_secret --secret-string <value>
+```
```

Comment:
Can you point to documentation where the user can find these secrets?

Reply:
added a link to azure plugin documenation